src/app/shared/comment-editor/comment-editor.component.ts

```diff
@@ -101,9 +102,18 @@ export class CommentEditorComponent implements OnInit {
     }
 
     const files = event.dataTransfer.files;
-    if (files.length > 0) {
-      this.readAndUploadFile(files[0]);
-      this.commentTextArea.nativeElement.focus();
+
+    if (files === undefined || files.length <= 0) {
+      return;
+    }
+
+    this.commentTextArea.nativeElement.focus();
+
+    for (let i = 0; i < files.length; i++) {
+      setTimeout(() => {
+        console.log(`File {i}. Begin uploading ${files[i].name}.`);
```

Comment:
Do remove console.log, or use the logging service.

Reply:
Ok thanks